src/Honeybee_Set EnergyPlus Zone Loads.py

```diff
@@ -29,9 +29,14 @@
         _HBZones: Honeybee zones for which you want to change the loads.
         equipmentLoadPerArea_: The desired equipment load per square meter of floor.  Values here should be in W/m2 (Watts per square meter).  Typical values can range from 2 W/m2 (for just a laptop or two in the zone) to 15 W/m2 for an office filled with computers and appliances.
         infiltrationRatePerArea_: The desired rate of outside air infiltration into the zone per square meter of floor.  Values here should be in m3/s-m2 (Cubic meters per second per square meter of floor).  ASHRAE recommends the following general infiltration rates based on the area of the facade exposed to the outdoors (note that you have to use the "Honeybee_infOrVentPerArea" to convert):
-            0.0001 (m3/s per m2 facade) - Tight building
-            0.0003 (m3/s per m2 facade) - Average building
-            0.0006 (m3/s per m2 facade) - Leaky building
+            ------------------------------------------------------------
+            Unit of following number: m3/s per m2 facade @75Pa
+            ------------------------------------------------------------
+            0.000047 - Passive house (0.1 cfm/sf facade @75Pa)
+            0.0001 - Tight building
+            0.000189 - ASHRAE 90.1-2013 (0.4 cfm/sf facade @75Pa)
+            0.0003 - Average building
+            0.0006 - Leaky building
```

Comment:
Thank you for adding the new numbers @MingboPeng . For clarification, have you done any conversion from 75 Pa pressure to typical building pressures in the numbers you have added here?

Reply:
Yes, those are numbers for 75Pa

Reply:
In that case, I think you need to make two separate lists here:
- One for the original values, which are meat to be at typical building pressures of ~4 Pa.
- Another for your new values that are meant to be at 75 Pa.

Actually, thinking through this further, I think it might be best to put these numbers on [this component](https://mostapharoudsari.gitbooks.io/honeybee-primer/text/components/infORventPerArea_Calculator.html), which is meant to calculate flow/m2 facade and has some built-in functions to approximate the pressure conversion from 75 to 4 Pa.

And one final note I'll make is that Honeybee[+] [is being set up to use flow/m2 facade for all infiltration values](https://github.com/ladybug-tools/honeybee-energy/blob/master/honeybee_energy/load/infiltration.py#L37-L42). So the whole headache of having to convert flow/m2 facade area to flow/m2 floor area is about to get some pain relief.

Reply:
Oh, right, I got those original numbers confused, I thought they were at 75Pa. 
I will try to adjust it in the legacy version. but for [+] version, could you please also add a note for those reference number is at 4Pa. https://discourse.ladybug.tools/t/recommended-infiltration-rate/1445/3?u=mingbopeng

Reply:
Thank you, Mingbo. I'll clarify in the Honeybee[+] docstring there that the numbers are meant to be at 4 Pa.

Reply:
Are you going to change the docstring here to make it clear which numbers are at 75 Pa vs. 4 Pa?